recipe/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
```

Comment:
As long as the recipe for x86 doesn't change, it's not necessary to bump the build number

Reply:
Actually, since we're going back to a version that already has released [builds](https://anaconda.org/conda-forge/xsimd/files) (though just with build_num=0), we should definitely bump here.